### ISSUE TITLE
Is it possible to customise Timeline axis labels

### ISSUE THREAD

**Issue by omkarbodke** [almende/vis#2417]
Is it possible to customise Timeline axis labels as, 
If the date range is between 15 days, then the axis will display in DD D format. 
If the date range is greater than 15 but less than a month, then the axis will display "|" for the day and DD D format for a weekday.

**Comment by yotamberk** [comment]
Hi @timelineAxis (wow... what a specific name lol),
take a look at: #2172
This is basically what you're looking for.

**Comment by timelineAxis** [comment]
Hi yotamberk, It was helpful, but for the week we able to format dates which are starting and ending dates of the week not the day which fall in between those (suppose step is 7).

**Comment by yotamberk** [comment]
not sure what you mean. Can you elaborate?

**Comment by timelineAxis** [comment]
@yotamberk :  Have a look at image, lines with black mark between two dates that's what we need. So, number of lines will be the days falls in between those two dates.
![image](https://cloud.githubusercontent.com/assets/24411932/21007895/1203d32a-bd65-11e6-91f3-9529a90bd8f7.png)

**Comment by yotamberk** [comment]
Okay I get it now. Unfortunately this is not possible yet =(
I would rephrase this issue to a new feature request.

**Comment by timelineAxis** [comment]
@yotamberk - Thanks

**Comment by mojoaxel** [comment]
@omkarbodke Maybe just zooming out could do the trick?

**Comment by yotamberk** [comment]
@omkarbodke closing this issue. If you want to submit a feature request with exactly what you need, please do!
